chromium_src/chrome/browser/certificate_manager_model.cc

```diff
@@ -133,8 +133,8 @@ void CertificateManagerModel::DidGetCertDBOnUIThread(
     CreationCallback callback) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
 
-  std::unique_ptr<CertificateManagerModel> model(
-      new CertificateManagerModel(cert_db, is_user_db_available));
+  auto model =
+      std::make_unique<CertificateManagerModel>(cert_db, is_user_db_available);
```

Comment:
This one was attempted in the previous issue but [caused the Linux build to fail](https://github.com/electron/electron/pull/29510#discussion_r646219650). You could change it to `base::WrapUnique` if you'd like, but I don't think it can use `std::make_unique` without some changes to `CertificateManagerModel` (would be good to reconcile it with any upstream changes in `chrome/browser/certificate_manager_model.h`).

Reply:
I had to rebuild locally and my build hasn't finished yet, that's why I didn't notice this one. Fixing now.